### ISSUE TITLE
Template inserting - autocomplete

### ISSUE THREAD

**Issue by michalk-k** [dbeaver/dbeaver#7163]
Can DBeaver offer template names during typing?
Let's say I created my template named: dosql
When I type it, I cannot see it is even recognized to be a template. Sometimes I may not remember exact name of the template, being able to enter only first letters. TAB works only if full tpl name is entered.
It would helpful to be assisted when typing template.

**Comment by kh** [comment]
Yes I would also like to be able to add template names in the autocomplete box (make it optional ?), it feels natural to have all suggestions possible.

![image](https://github.com/dbeaver/dbeaver/assets/31128/3a2fab2d-2eaa-4ea0-83ec-a02acbe73cfa)

![image](https://github.com/dbeaver/dbeaver/assets/31128/0d3a0871-5613-4809-8a3b-81e5d9d3d0ad)


![image](https://github.com/dbeaver/dbeaver/assets/31128/d0b55698-318d-4dad-bc01-376b32c4e94a)
